packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1695,7 +1700,8 @@ const VideoPlayerBase = class extends React.Component {
 	getControlsAriaProps () {
 		if (this.state.announce === AnnounceState.TITLE) {
 			return {
-				role: 'region',
+				role: 'alert',
+				'aria-live': 'off',
 				'aria-labelledby': `${this.id}_title`
```

Comment:
Please order the properties in the object above.

Reply:
Done.